src/Shared/PropertyHelper/PropertyHelper.cs

```diff
@@ -212,8 +212,6 @@ public void SetValue(object instance, object? value)
         // MakeGenericMethod + value type requires IsDynamicCodeSupported to be true.
         if (RuntimeFeature.IsDynamicCodeSupported)
         {
-            // TODO: Remove disable when https://github.com/dotnet/linker/issues/2715 is complete.
-#pragma warning disable IL3050 // Calling members annotated with 'RequiresDynamicCodeAttribute' may break functionality when AOT compiling.
```

Comment:
There are a few more places, can we remove them as well?

https://github.com/dotnet/aspnetcore/blob/12e4c649c37e61ea34be2b4b86dc5c7473860fe5/src/Http/Routing/src/Matching/JumpTableBuilder.cs#L92-L93

https://github.com/dotnet/aspnetcore/blob/12e4c649c37e61ea34be2b4b86dc5c7473860fe5/src/Http/Http.Results/src/ResultsOfTHelper.cs#L50-L51

Reply:
Hooray for vscode.dev!